swagger.json

```
@@ -404,6 +430,43 @@
           }
         }
       }
+    },
+    "/location": {
+      "post": {
+        "x-swagger-router-controller": "location",
+        "operationId": "getLocation",
+        "summary": "Gets the location of the user given a fingerprint and a building Id",
+        "tags": [
+          "/location"
+        ],
+        "security": [
+          {
+            "ApiKey": []
+          }
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "fingerprint",
+            "description": "Location body with fingerprint and id",
+            "schema": {
+              "$ref": "#/definitions/FingerprintWithId"
```

```suggestion
              "$ref": "#/definitions/Fingerprint"
```

Cuando pedís una location no le mandás una fingerprint que ya existe (o sea con ID), armás una vos